docs/guide/content/development_guide/connectors.md

```diff
@@ -0,0 +1,40 @@
+# Connectors
+
+Connectors are a key extension point of OnDemand Loop. They abstract repository specific
```

Comment:
```suggestion
Connectors are a key extension point of [OnDemand Loop](https://github.com/IQSS/ondemand-loop). They abstract repository specific
```

Reply:
I do not agree that every single `OnDemand Loop` should be a link to the repo